_examples/large-project-structure/integration/go.mod

```
@@ -0,0 +1,17 @@
+module github.com/corelight/integration
```

Do you mind if we rename this something that matches the directory like:
```suggestion
module github.com/99designs/gqlgen/_examples/large-project-structure/integration
```